src/java.desktop/share/classes/java/awt/color/ICC_Profile.java

```diff
@@ -963,12 +965,7 @@ public static ICC_Profile getInstance(String fileName) throws IOException {
      *         Profile data
      */
     public static ICC_Profile getInstance(InputStream s) throws IOException {
-        byte[] profileData;
-        if ((profileData = getProfileDataFromStream(s)) == null) {
```

Comment:
This additional check was needed when the getInstance(byte[]) thrown NPE on null